phpcs.xml.dist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<ruleset name="PHP_CodeSniffer">
+ <description>Default configuration</description>
+
+ <rule ref="vendor/cakephp/cakephp-codesniffer/CakePHP"/>
+
+ <file>./src</file>
+ <file>./tests</file>
+
+ <arg phpcs-only="true" value="p"/>
+ <arg name="extensions" value="php"/>
+
+ <config name="colors" value="1"/>
+</ruleset>
```

Comment:
Ah, okay. Fixed.
Didn't know that it is matter to xml files too. Now i searched about that and founded the [explanation](http://stackoverflow.com/a/729795/3653002) why.